### ISSUE TITLE
FMC not load

### ISSUE THREAD

**Issue by RfadnjdExt** [gefs-plugins/fmc-requirejs#11]
Theres no FMC button.
Console :
```[Deprecation] document.registerElement is deprecated and will be removed in M73, around March 2019. Please use window.customElements.define instead. See https://www.chromestatus.com/features/4642138092470272 for more details.
(anonymous) @ index.html:22
(anonymous) @ index.html:22
VM89:1 [Deprecation] 'webkitURL' is deprecated. Please use 'URL' instead.
(anonymous) @ VM89:1
K @ VM49:3
Z.D @ VM49:3
L @ VM49:3
y @ VM49:3
x @ VM49:3
A @ VM49:3
Z @ VM49:3
l.YKR @ VM49:3
l.bg @ VM49:3
ib.za @ addata.js:1
load (async)
db @ addata.js:1
ib @ addata.js:1
K @ addata.js:1
(anonymous) @ addata.js:1
xa @ addata.js:1
(anonymous) @ addata.js:1
Cesium.js:451 The Scene.fxaa property has been deprecated. Use Scene.postProcessStages.fxaa.
i @ Cesium.js:451
i @ Cesium.js:451
set @ Cesium.js:532
geofs.api.renderingQuality @ geofs.js?kc=1534504809_5:85
(anonymous) @ geofs.js?kc=1534504809_5:174
geofs.readPreferences @ geofs.js?kc=1534504809_5:197
geofs.init @ geofs.js?kc=1534504809_5:174
(anonymous) @ geofs.js?kc=1534504809_5:189
window.fireBasicEvent @ loader.js:36
loadDeferredScript @ loader.js:68
script.onload @ loader.js:77
load (async)
loadDeferredScript @ loader.js:75
script.onload @ loader.js:77
load (async)
loadDeferredScript @ loader.js:75
script.onload @ loader.js:77
load (async)
loadDeferredScript @ loader.js:75
(anonymous) @ loader.js:83
load (async)
(anonymous) @ loader.js:60
geofs.js?kc=1534504809_5:522 The Web Audio autoplay policy will be re-enabled in Chrome 71 (December 2018). Please check that your website is compatible with it. https://goo.gl/7K7WLu
createPlayer @ geofs.js?kc=1534504809_5:522
audio.init @ geofs.js?kc=1534504809_5:514
geofs.aircraft.Aircraft.init @ geofs.js?kc=1534504809_5:329
success @ geofs.js?kc=1534504809_5:325
i @ commonJS.js?kc=1536173784:2
fireWith @ commonJS.js?kc=1536173784:2
y @ commonJS.js?kc=1536173784:4
c @ commonJS.js?kc=1536173784:4
XMLHttpRequest.send (async)
send @ commonJS.js?kc=1536173784:4
ajax @ commonJS.js?kc=1536173784:4
geofs.aircraft.Aircraft.load @ geofs.js?kc=1534504809_5:325
(anonymous) @ geofs.js?kc=1534504809_5:176
geofs.readPreferences @ geofs.js?kc=1534504809_5:197
geofs.init @ geofs.js?kc=1534504809_5:174
(anonymous) @ geofs.js?kc=1534504809_5:189
window.fireBasicEvent @ loader.js:36
loadDeferredScript @ loader.js:68
script.onload @ loader.js:77
load (async)
loadDeferredScript @ loader.js:75
script.onload @ loader.js:77
load (async)
loadDeferredScript @ loader.js:75
script.onload @ loader.js:77
load (async)
loadDeferredScript @ loader.js:75
(anonymous) @ loader.js:83
load (async)
(anonymous) @ loader.js:60
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/9/285/326.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 285 Y: 185 Level: 9.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/10/570/654.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/10/570/655.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 570 Y: 369 Level: 10.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 570 Y: 368 Level: 10.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/10/571/654.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/10/569/653.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 571 Y: 369 Level: 10.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 569 Y: 370 Level: 10.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1140/1308.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1141/1309.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1140/1309.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1140 Y: 739 Level: 11.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1141 Y: 738 Level: 11.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1140 Y: 738 Level: 11.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1139/1309.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1139/1308.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1139 Y: 738 Level: 11.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1139 Y: 739 Level: 11.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2280/2618.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2280/2619.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2279/2618.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2280 Y: 1477 Level: 12.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2280 Y: 1476 Level: 12.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2279 Y: 1477 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4559/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4560/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4560/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4559 Y: 2954 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4560 Y: 2955 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4560 Y: 2954 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4558/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4558/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4559/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4558 Y: 2955 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4558 Y: 2954 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4559 Y: 2955 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4561/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4561/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4559/5238.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4561 Y: 2954 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4561 Y: 2955 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4559 Y: 2953 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4558/5238.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2279/2619.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2281/2618.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4558 Y: 2953 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2279 Y: 1476 Level: 12.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2281 Y: 1477 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4557/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4557/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2281/2619.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4557 Y: 2955 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4557 Y: 2954 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2281 Y: 1476 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4558/5239.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4559/5239.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2279/2617.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4558 Y: 2952 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4559 Y: 2952 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2279 Y: 1478 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2278/2618.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4557/5238.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2278/2619.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2278 Y: 1477 Level: 12.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4557 Y: 2953 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2278 Y: 1476 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2278/2616.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2277/2616.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2277/2617.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2278 Y: 1479 Level: 12.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2277 Y: 1479 Level: 12.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2277 Y: 1478 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4557/5239.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2278/2617.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4557 Y: 2952 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2278 Y: 1478 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4559/5235.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4559 Y: 2956 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4558/5235.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4559/5234.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4557/5235.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4558 Y: 2956 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4559 Y: 2957 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4557 Y: 2956 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4558/5234.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4557/5234.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4558 Y: 2957 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4557 Y: 2957 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4556/5235.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4556 Y: 2956 Level: 13.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4556/5234.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1141/1308.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2279/2616.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4556 Y: 2957 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1141 Y: 739 Level: 11.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2279 Y: 1479 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2277/2618.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2277 Y: 1477 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1139/1307.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1139 Y: 740 Level: 11.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1138/1307.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/12/2276/2616.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1138 Y: 740 Level: 11.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 2276 Y: 1479 Level: 12.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/10/569/652.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1139/1306.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/11/1138/1306.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 569 Y: 371 Level: 10.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1139 Y: 741 Level: 11.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 1138 Y: 741 Level: 11.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/10/568/652.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 568 Y: 371 Level: 10.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4555/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4554/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4554/5236.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4555/5238.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
geofs.php:1 Access to image at 'https://tiles.geo-fs.com/13/4555/5237.jpg' from origin 'https://www.geo-fs.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4555 Y: 2955 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4554 Y: 2954 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4554 Y: 2955 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4555 Y: 2953 Level: 13.
Cesium.js:459 An error occurred in "v": Failed to obtain image tile X: 4555 Y: 2954 Level: 13.```

**Comment by RfadnjdExt** [comment]
![image](https://user-images.githubusercontent.com/40250666/48418710-894ea000-e788-11e8-9640-733ffd13af79.png)

**Comment by harryxue1999** [comment]
Issue closed. Duplicate of #10.

**Comment by hAr1x** [comment]
Duplicate of #10
